segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBConfig.java

```diff
@@ -69,6 +70,13 @@
     @Getter
     private final boolean directReads;
 
+    /**
+     * Enabling memory only will let rocksdb write everything into memory instead of disks. This supposed to provide
+     * very high performance since there is not disk IO incurred.
+     */
+    @Getter
+    private final boolean memoryOnly;
```

Comment:
We need to add this property and the default value in the `config.properties` file.

Reply:
Added to the properties. Thanks!